playbooks/openshift/roles/bfs/tasks/clone_from_github.yml

```diff
@@ -11,4 +15,5 @@
     clone: yes
     update: yes
     accept_hostkey: yes
-  when: dot_git_path.stat.isdir is undefined or dot_git_path.stat.isdir == False
+  when: (dot_git_path.stat.isdir is undefined) or
```

Comment:
Why did you decide to use parens here and not elsewhere?

Reply:
i find it easier to use parens to put in the spot light the `or` condition due to the fact that ansible doesn't let me do something like this

```
  when: 
    - dot_git_path.stat.isdir is undefined or
    - dot_git_path.stat.isdir == False
```
nor
```
  when: 
    - (dot_git_path.stat.isdir is undefined) or
    - (dot_git_path.stat.isdir == False)
```
however is a different story for  `and` condition where when you can do 

```
  when:
    - git_log.stdout == ""
    - bleeding_edge | bool == false
```
where `and` is implicit and you can't miss it